.circleci/config.yml

```diff
@@ -720,6 +720,61 @@ jobs:
       - run:
           name: test
           command: cd core && ./node_modules/.bin/jest __tests__/notifiers --ci --maxWorkers 2
+  test-core-ops:
```

Comment:
You could move this new test into a model test too if you don't want to start the whole ops section